sys/include/net/gnrc/lorawan/region.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup net_gnrc_lorawan
+ * @{
+ *
+ * @file
+ * @brief   GNRC LoRaWAN region specific functions
+ *
+ * @author  José Ignacio Alamos <jose.alamos@haw-hamburg.de>
+ */
+#ifndef NET_GNRC_LORAWAN_REGION_H
+#define NET_GNRC_LORAWAN_REGION_H
+
+#include "net/gnrc/lorawan.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+#define GNRC_LORAWAN_LC_1 (868100000)   /**< First default EU868 channel */
+#define GNRC_LORAWAN_LC_2 (868300000)   /**< Second default EU868 channel */
+#define GNRC_LORAWAN_LC_3 (868500000)   /**< Third default EU868 channel */
+
+#define GNRC_LORAWAN_DEFAULT_CHANNELS 3 /**< Number of default channels for the current region */
```

Comment:
this should be named `GNRC_LORAWAN_DEFAULT_CHANNELS_NUMOF` to be consistent with other similar names in RIOT like `I2C_NUMOF` and so on

Reply:
and using an array as suggested above this will be:
```
#define GNRC_LORAWAN_DEFAULT_CHANNELS_NUMOF (sizeof(gnrc_lorawan_default_channels)/sizeof(uint32_t))
```

Reply:
done